modules/sff/module/src/sff_enums.c

```diff
@@ -905,6 +908,9 @@ aim_map_si_t sff_module_type_desc_map[] =
     { "1GBASE-T", SFF_MODULE_TYPE_1G_BASE_T },
     { "100BASE-LX", SFF_MODULE_TYPE_100_BASE_LX },
     { "100BASE-FX", SFF_MODULE_TYPE_100_BASE_FX },
+    { "100GBASE-LR", SFF_MODULE_TYPE_100G_BASE_LR},
```

Comment:
You should group the 100G enums together.

Reply:
grouped 100G...